packages/plugin-interaction-breadcrumbs/src/interaction-breadcrumbs.ts

```diff
@@ -1,10 +1,13 @@
+import { Plugin } from '@bugsnag/core'
+import type ClientWithInternals from 'packages/core/client'
```

Comment:
so this import is working fine?

Reply:
This one happens to work fine because that import path doesn't appear in the final generated types:

```